scripts/docker_files/docker_file_ci_centos_7_python35/DockerFile

```
@@ -0,0 +1,44 @@
+FROM centos:centos7
+
+USER root
+
+ENV HOME /root
+
+#RUN python --version
+RUN yum -y --quiet install wget git cmake gcc-c++ unzip python-pip lapack-devel make gcc-gfortran zlib-devel lapack-devel bzip2-devel openssl-devel samba-client openmpi-devel hdf5-openmpi-devel
```

```suggestion
RUN yum -y --quiet install wget git cmake gcc-c++ unzip python-pip make gcc-gfortran zlib-devel bzip2-devel openssl-devel samba-client openmpi-devel hdf5-openmpi-devel
```

we don't use lapack (any more)
also was duplicated